src/test/java/org/mifos/integrationtest/cucumber/stepdef/PayerFundTransferStepDef.java

```diff
@@ -186,6 +188,42 @@ public void callCreateInteropIdentifierEndpoint(String client) throws JsonProces
         assertThat(fundTransferDef.responseInteropIdentifier).isNotEmpty();
     }
 
+    @Then("I call the interop identifier endpoint for {string} and accountId {string}")
+    public void callCreateInteropBudgetIdentifierEndpoint(String client, String accountId) throws JsonProcessingException {
+        // Setting headers and body
+        RequestSpecification requestSpec = Utils.getDefaultSpec();
+        requestSpec = fundTransferDef.setHeaders(requestSpec);
+        fundTransferDef.interopIdentifierBody = fundTransferDef.setBodyInteropIdentifier();
+        // Setting account ID in path
+
+        String responseSavingsAccount = client.equals("payer") ? fundTransferDef.responseSavingsAccountPayer
+                : fundTransferDef.responseSavingsAccountPayee;
+
+        PostSavingsAccountsResponse savingsAccountResponse = objectMapper.readValue(responseSavingsAccount,
+                PostSavingsAccountsResponse.class);
+        savings_account_id = savingsAccountResponse.getSavingsId().toString();
+
+        if (client.equals("payer")) {
+            payer_identifier = accountId;
+            scenarioScopeState.payerIdentifier = accountId;
+        } else {
+            payee_identifier = accountId;
+            scenarioScopeState.payeeIdentifier = accountId;
+        }
+
+        String endpoint = transferConfig.interopIdentifierEndpoint;
+        endpoint = endpoint.replaceAll("\\{\\{identifierType\\}\\}", "MSISDN");
```

Comment:
use  string.format()

Reply:
suggestion: find a better approach to do the placeholder replacement. write a util function for that